jest_setup.js

```diff
@@ -0,0 +1,17 @@
+// Jetpack mocks
+global.window.JP_CONNECTION_INITIAL_STATE = {};
```

Comment:
This was added to remove the console error: 

`Jetpack Connection package: Initial state is missing. `

This was happening before the Jest upgrade.

Reply:
I saw this approach is also used in the Jetpack unit tests: https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/connection/jest.setup.js

Reply:
I tried to use this setup but it looks like it is not compatible with the environment we use:

```javascript
TypeError: Cannot convert undefined or null to object
        at Function.getOwnPropertyDescriptors (<anonymous>)

      window.location = Object.defineProperties(
```

What do you think if we copy these:

```javascript
window.JP_CONNECTION_INITIAL_STATE = {
	userConnectionData: {
		currentUser: {
			wpcomUser: { Id: 99999, login: 'bobsacramento', display_name: 'Bob Sacrmaneto' },
		},
	},
};
```

Into our `global.window.JP_CONNECTION_INITIAL_STATE` mock?

Reply:
Something like this https://github.com/wordpress-mobile/gutenberg-mobile/pull/5544/commits/29521810d3368f3f0a6b45f07a66cf8d42d78412

Reply:
Yeah, that approach looks good to me, thanks! In the future, we'd need to revisit how to manage the Jetpack connection in GB-mobile and populate with data coming form the host (i.e. the app).